code/modules/vending/wardrobes.dm

```diff
@@ -525,7 +525,7 @@
 					/obj/item/clothing/shoes/laceup = 2,
 					/obj/item/clothing/suit/det_suit/grey = 1,
 					/obj/item/clothing/suit/det_suit/noir = 1,
-					/obj/item/clothing/head/fedora = 2,
+					/obj/item/clothing/head/fedora/det_hat/noir = 2,
```

Comment:
Current change removes the default brown fedora from the vendor
```suggestion
					/obj/item/clothing/head/fedora/det_hat = 2,
					/obj/item/clothing/head/fedora/det_hat/noir = 2
```

Reply:
It does not, the original detective hat is further up the list of products

Reply:
Ah you're totally right, you should keep the 2 hats next to each other regardless for the sake of them showing up next to each other in the vendor

Reply:
They're grouped by outfit theme. Like the original fedora is with the brown trenchcoat, shoes, and suit, and the noir fedora is with the noir clothing and black laceup shoes.

Reply:
Alright, then code's solid